src/RoleObjects/CellData.js

```diff
@@ -22,30 +15,28 @@ export default class CellData extends SectionData {
    * Gives an element's column index or position with respect to the total
    * number of columns within a table, grid, or treegrid.
    * @access public
-   * @returns {Number} - Positive number
+   * @returns {Number} - column index.  undefined if the field is unset
    */
   get colindex() {
     return this._reactProps['aria-colindex'];
   }
 
   /**
    * Sets the number of columns spanned by a cell or gridcell within a table, grid, or treegrid.
-   * This value must be at least 1 (default) and low enough to avoid overlap with other
-   * cells in the same row
    * @access public
-   * @param {Number} val - number
+   * @param {Number} val - number of columns spanned by this cell.  undefined to unset the field
    */
   set colspan(val) {
-    this._reactProps['aria-colspan'] = val;
+    this._reactProps.colspan = val;
```

Comment:
Non-aria attributes should be preferred over aria-* ones and since this role uses the semantic td tag, that means just `colspan` and `rowspan` for this role